src/App.native.tsx

```diff
@@ -201,7 +201,7 @@ function App() {
   return (
     <GeolocationProvider>
       <A11yProvider>
-        <KeyboardControllerProvider>
+        <KeyboardProvider enabled={true}>
```

Comment:
Disabling `react-native-keyboard-controller` (`enabled={false}` on the provider) actually breaks edge-to-edge as it will call `WindowCompat.setDecorFitsSystemWindows(window, true)`. See https://github.com/kirillzyusko/react-native-keyboard-controller/pull/765 for more infos / understand the potential fix.